gluten-core/src/main/scala/io/glutenproject/extension/ColumnarOverrides.scala

```diff
@@ -572,6 +572,19 @@ case class TransformPostOverrides(session: SparkSession, isAdaptiveContext: Bool
       p.withNewChildren(children)
   }
 
+  def nativeSupportedPlan(child: SparkPlan): Boolean = {
+    if (child.isInstanceOf[TransformSupport]) {
+      return true
+    }
+    if (!child.supportsColumnar) {
```

Comment:
As we know, `ColumnarToRow` is inserted by spark with child plan checked like this line. So I think we can remove the redundant check here. Right?